web/app/controllers/deployments_controller.rb

```diff
@@ -1,8 +1,9 @@
 require 'deployment_pb'
 
 class DeploymentsController < ApplicationController
-  before_action :set_application, only: [:index, :create, :new, :show]
-  before_action :set_environments, only: [:new, :create]
+  before_action :set_application, only: %i[index create new show]
+  before_action :set_environments, only: %i[new create]
+  before_action :find_commits, only: %i[new create]
```

Comment:
Does the `create` action need the commits populated? Seems like only the view generated by the `new` action needs them. Would save a bit of time when creating a deploy.

Reply:
Yeah, that's a good point, I will only do it for new then.